modules/cache-common/src/test/java/org/opensearch/cache/common/tier/TieredSpilloverCacheTests.java

```diff
@@ -1501,6 +1760,67 @@ private ImmutableCacheStats getStatsSnapshotForTier(TieredSpilloverCache<?, ?> t
         return snapshot;
     }
 
+    @SuppressWarnings({ "rawtypes", "unchecked" })
+    private void verifyComputeIfAbsentThrowsException(
+        Class<? extends Exception> expectedException,
+        LoadAwareCacheLoader loader,
```

Comment:
you can remove these warnings by providing the type of `LoadAwareCacheLoader<ICacheKey<String>, String>`

Reply:
Yeah I somehow missed this.